src/diffusers/image_processor.py

```diff
@@ -68,6 +72,18 @@ def numpy_to_pil(images):
 
         return pil_images
 
+    @staticmethod
+    def pil_to_numpy(images):
+        """
+        Convert a PIL image or a list of PIL images to numpy image
```

Comment:
```suggestion
        Convert a PIL image or a list of PIL images to numpy arrays.
```

Can we also have type annotations here?